GomokuClient/packages/gomoku-core/src/features/GameTime/mobile/GameTimeMobile.tsx

```diff
@@ -51,13 +55,15 @@ export const GameTimeMobile = ({
         <div className="flex items-center">
           <span
             className="mr-2 h-3 w-3 rounded-full"
-            style={{ backgroundColor: player.color }}
+            style={{ backgroundColor: player?.color }}
           ></span>
-          <span className="font-bold text-[#ffa600]">{player.name}</span>
-        </div>
-        <div className="bg-[#3d3733] px-2 py-1 text-4xl font-bold">
-          {blackTimeLeft || whiteTimeLeft}
+          <span className="font-bold text-[#ffa600]">{player?.userName}</span>
         </div>
+        {timeLeft && (
+          <div className="bg-[#3d3733] px-2 py-1 text-4xl font-bold">
+            {secondsToString(timeLeft)}
```

Comment:
I don't see this component is now functioning, can you please make adjustments?

<img width="608" alt="Screenshot 2024-11-06 at 20 06 46" src="https://github.com/user-attachments/assets/dfe410af-b24a-4f72-9c4b-5caaaf3a926f">

Reply:
Works on my machine

![image](https://github.com/user-attachments/assets/f3960e9a-6fce-43a1-b49d-2f873c60afcc)